presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1000,4 +1003,30 @@ public HiveClientConfig setHdfsPrestoKeytab(String hdfsPrestoKeytab)
         this.hdfsPrestoKeytab = hdfsPrestoKeytab;
         return this;
     }
+
+    public boolean isBucketExecutionEnabled()
+    {
+        return bucketExecutionEnabled;
+    }
+
+    @Config("experimental.hive.bucket-execution")
```

Comment:
I'm not sure about the names of these config properties.  I would get @electrum  to review these.  Specifically, I don't know if we want to prefix with `experimental` or even include it at all.